lib/constants/validators.dart

```diff
@@ -54,3 +54,18 @@ Function(String) bloodGroupValidator = (String bloodGroup) {
 
   return null;
 };
+
+Function(String) emailValidator = (String emailaddress) {
+  String email = r"^[a-zA-Z0-9.a-zA-Z0-9.!#$%&'*+-/=?^_`{|}~]+@[a-zA-Z0-9]+\.[a-zA-Z]+";
+  bool isValid = RegExp(email).hasMatch(emailaddress);
```

Comment:
rename this variable as `emailAddress`. Follow camelCasing for naming variables :)